docs/_user_guide/storage.md

```diff
@@ -1,276 +1,180 @@
 
-**Backups**
+!!! Note
+    To create a new faculty or student project space, please complete the project space request form [here](http://darc.stanford.edu/Yenstorage/){:target="_blank"}. If you would like to discuss specific storage solutions for your project, Please email the [DARC](mailto:gsb_darcresearch@stanford.edu) team to discuss further.
 
-Files on ZFS are backed up as "snapshots" and can be restored manually by system admins. Please see the page [How Do I Recover ZFS Files](/faqs/howRecoverZFSFiles.html) for instructions on recovering files. We also maintain an off-site disaster recovery solution for both ZFS and home directories.
+### Scratch Space
 
-### Yen Home Directories
-
-Home directories on the Yen servers are also stored on our storage system. Your home directory has a quota of 50 GB.
-
-{% include tip.html content="To check your home directory quota, use the `gsbquota` command"%}
-
-When you log into the Yen servers you will automatically land in your home directory, which is located at ```/home/users/{SUNet-id}``` or with the shortcut `~`. You are able to access your former AFS home directory by following the ```afs-home``` symlink inside your home directory. 
-
-{% include important.html content="Please note that if you continue to work and save files within your older afs-home directory you will still be limited to the old AFS disk quota (5 GB)" %}
-
-### Sratch Space 
-There is a large ZFS based scratch space, accessible from any yen, at ```/scratch/shared```, which is 100 T in size. Reads and writes to this space are slower than to local disk.
+There is a large ZFS based scratch space, accessible from any Yen, at ```/scratch/shared```, which is 100 TB in size. Reads and writes to this space are slower than to local disk.
 
 ### Local Disk
 
-On each Yen machine, there is local disk space mounted at ```/tmp```, which is over 1 T in size. Reading and writing from ```/tmp``` is a lot faster because I/O operations do not have to go via the slow network. All yen users are free to make use of this space. Much like a hard drive on your laptop, this can be accessed only from that single Yen machine. Be careful not to fill up `/tmp` completely as jobs running on that yen node may crash in unexpected ways. Be a good citizen and delete the files you no longer need. 
+On each Yen machine, there is local disk space mounted at ```/tmp```, which is over 1 TB in size. Reading and writing from ```/tmp``` is a lot faster because I/O operations do not have to go via the slow network. All Yen users are free to make use of this space. Much like a hard drive on your laptop, this can be accessed only from that single Yen machine. Be careful not to fill up `/tmp` completely as jobs running on that Yen node may crash in unexpected ways. Be a good citizen and delete the files you no longer need.
```

Comment:
Should we mention that you should be create /tmp/$USER/ for your own tmp space to not confuse with other people tmp spaces?

Reply:
Following Alex's suggestion, I added a sentence reminding users to ensure that their files in `/tmp` have the correct permissions set to prevent unauthorized access or manipulation.